tokio-console/src/main.rs

```diff
@@ -95,6 +95,11 @@ async fn main() -> color_eyre::Result<()> {
                 let input = input
                     .ok_or_else(|| eyre!("keyboard input stream ended early"))
                     .with_section(|| "this is probably a bug".header("Note:"))??;
+
+                if input::should_ignore_key_event(&input) {
```

Comment:
https://github.com/tokio-rs/console/blob/bb463c1b80f0cdaa639c547f397f1102da259c9c/tokio-console/src/main.rs#L81

We already have a filter here to ignore the release event. Maybe we should consider merging them together.

Reply:
I missed seeing that, fixing it by just using the should_ignore_key_event instead of the filter. Either way would be fine I guess.

Reply:
@hds @hawkw Do you have any thoughts about which one is better? It seems both work well. The only difference is that the try_filter will filter it asynchronously.